base/applications/cmdutils/tasklist/lang/ro-RO.rc

```diff
@@ -1,20 +1,27 @@
+/*
+ * PROJECT:     ReactOS Tasklist
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     The tool shows the list of tasks
+ * COPYRIGHT:   Copyright 2023 Andrei Miloiu <miloiuandrei@gmail.com>
+ */
+
 LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
 
 STRINGTABLE
 BEGIN
-    IDS_USAGE, "Usage: TASKLIST [/?] [/NH]\n"
+    IDS_USAGE, "Utilizare: TASKLIST [/?] [/NH]\n"
     IDS_DESCRIPTION, "\n\
-Desciption:\
-    This tool displays a list of currently running processes on\n\
-    the machine.\n"
-    IDS_OPTION_TOO_MUCH, "Error: Invalid syntax. '%1' option is not allowed more than '%2!d!' time(s).\n"
-    IDS_INVALID_SYNTAX, "Error: Invalid syntax.\n"
-    IDS_INVALID_OPTION, "Error: Unknown or invalid command line option specified.\n"
-    IDS_ENUM_FAILED, "Error: Unable to enumerate the process list.\n"
-    IDS_OUT_OF_MEMORY, "Error: Out of memory.\n"
-    IDS_HEADER_IMAGENAME, "Image Name"
+Descriere:\
+    Această unealtă afișează o listă a proceselor curente care rulează pe\n\
+    calculator.\n"
```

Comment:
Just two bits (that are not a request for change):
- in my work I think I was translating (English) "tool" as "instrument" (adapted) in Romanian; so (for consistency's sake) please remember to update the other translations if you stick to "unealtă"
- "care rulează pe" can be reduced to "din" and it will be fine (but that's just a matter of taste)

Reply:
I fixed it. Please recheck with the last commit.